src/aiida/cmdline/params/options/main.py

```diff
@@ -743,3 +752,56 @@ def set_log_level(_ctx, _param, value):
     is_flag=True,
     help='Print the full traceback in case an exception is raised.',
 )
+
+PATH = OverridableOption(
```

Comment:
Now that there is just one command using these, it doesn't make a whole lot of sense to keep these options here as reusable ones. Especially since they are really quite specific (look at the help strings which only really make sense in the context of the dump command). Do we not just want to move this to the `cmd_process.py` itself?

Reply:
Yeah, I also considered this, but decided against it, as dumping larger entities (e.g. groups or a whole profile archive) is also on the todo-list, and I anticipate that I'll re-use these options there. That's why I didn't move them to `cmd_process.py` as normal `click` options. Is that fine?

Reply:
Bit premature I would say, and I think the help strings should be generalized if you keep them here (you can make them more specific when you actually use the option in the command), but ok to leave them here for now then

Reply:
Fair enough! I moved `INCLUDE_INPUTS`, `INCLUDE_OUTPUTS`, `INCLUDE_ATTRIBUTES`, `INCLUDE_EXTRAS`, and `FLAT` to `cmd_process` as `click` options, and left `PATH` and `OVERWRITE` in `main.py` with more general descriptions :-)